tools/serverpod_cli/lib/src/generator/dart/entities_library_generator.dart

```diff
@@ -1,49 +1,33 @@
 import 'package:built_collection/built_collection.dart';
 import 'package:code_builder/code_builder.dart';
+import 'package:serverpod_cli/analyzer.dart';
+import '../shared.dart';
 
-import 'class_generator.dart';
-import 'config.dart';
-import 'protocol_definition.dart';
-import 'types.dart';
+/// Generates the dart libraries for [SerializableEntityDefinition]s.
+class EntityLibraryGenerator {
+  final bool serverCode;
+  final GeneratorConfig config;
 
-String serverpodUrl(bool serverCode) {
-  return serverCode
-      ? 'package:serverpod/serverpod.dart'
-      : 'package:serverpod_client/serverpod_client.dart';
-}
-
-String serverpodProtocolUrl(bool serverCode) {
-  return serverCode
-      ? 'package:serverpod/protocol.dart'
-      : 'package:serverpod_client/serverpod_client.dart';
-}
-
-class ClassGeneratorDart extends ClassGenerator {
-  @override
-  String get outputExtension => '.dart';
-
-  ClassGeneratorDart({
-    required super.outputDirectoryPath,
-    required super.verbose,
-    required super.serverCode,
-    required super.classDefinitions,
-    required super.protocolDefinition,
+  EntityLibraryGenerator({
```

Comment:
`SerializableEntityLibraryGenerator` to keep things consistent

Reply:
46b26fb8